packages/calypso-e2e/src/browser-manager.ts

```diff
@@ -200,6 +200,21 @@ export async function setLoginCookie( page: Page, accountType: string ): Promise
 		throw new Error( `Cookie file ${ cookiePath } not found on disk.` );
 	}
 
+	/*
+	Check the creation time of the cookie.
+	WordPress.com cookies are valid for a limited time, typically less than 3 days.
+	So if a cookie file is found, but the creation time is older than 3 days, pretend that
+	a cookie file did not exist.
+	This is only an issue when running the tests locally.
+	*/
```

Comment:
Let's use this format for inline comments only (`/* inline comment */`), and use `//` for multiline ones to stay consistent across the codebase. https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#comments

Reply:
Interesting, I thought it was the other way around - admittedly my last experience with Java-syntax languages (before A8C) was in university with Java where I used block comments for multi-line comments. 

I will change the comment format but I still think a block comment is better from a formatting perspective.

Reply:
For multiline comments, I believe the other popular standard (and the one used in WP *.php files) is the starred-block:

```js
/**
 * Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec lectus augue,
 * aliquam dapibus est eget, posuere ornare sem. Quisque nec cursus elit.
 * Maecenas egestas at nunc nec cursus. Curabitur efficitur id ex at pharetra.
 * Nam turpis risus, laoreet quis odio eget, mollis sodales justo. Morbi commodo
 * sem dignissim, porttitor ipsum ac, lobortis nulla.
 */
```

Reply:
Oh duh, I totally forgot about the second star in the first line.

I forgot to push the changes here before I (overzealously) hit the merge button, so I've pushed the changes to https://github.com/Automattic/wp-calypso/pull/58728 and merged it together.